src/chrome/content/rules/Phoronix-Media.xml

```diff
@@ -1,25 +1,31 @@
 <!--
 	Nonfunctional:
 
+		- (www.)phoronix.net ¹
+		- (www.)phoronix-media.com ¹
 		- (www.)phoromatic.com ¹
-		- (www.)phoronix.com	(ssl_error_rx_record_too_long)
-
-	¹ Shows blank page; self-signed, CN: phx1.phoronix.com
+		- (www.)phoronix-test-suite.com ¹
+		- (www.)phoronix.com/forums (mixed content)
 
+	¹ ssl_error_rx_record_too_long
 -->
-<ruleset name="Phoronix Media (partial)" platform="mixedcontent">
 
+<ruleset name="Phoronix Media">
+  	<target host="phoronix.com" />
+  	<target host="www.phoronix.com" />
 	<target host="openbenchmarking.org" />
 	<target host="www.openbenchmarking.org" />
 
+	<exclusion pattern="^http://(www\.)?phoronix\.com/forums/" />
+		<test url="http://phoronix.com/forums/" />
+		<test url="https://phoronix.com/forums/" />
+		<test url="http://www.phoronix.com/forums/" />
+		<test url="https://www.phoronix.com/forums/" />
+  	<rule from="^https://(www\.)?phoronix\.com/forums/" to="http://$1phoronix.com/forums/" downgrade="1" />
```

Comment:
Can you confirm this downgrade is necessary? I.e., are there pages on phoronix.com that point incorrectly to the HTTPS version of the forums, causing breakage without it?